Sources/Sentry/SentryReplayRecording.m

```diff
@@ -0,0 +1,64 @@
+#import "SentryReplayRecording.h"
+#import "SentryDateUtil.h"
+
+@implementation SentryReplayRecording
+
+- (instancetype)initWithSegmentId:(NSInteger)segmentId
+                             size:(NSInteger)size
+                            start:(NSDate *)start
+                         duration:(NSTimeInterval)duration
+                       frameCount:(NSInteger)frameCount
+                        frameRate:(NSInteger)frameRate
+                           height:(NSInteger)height
+                            width:(NSInteger)width
+{
+    if (self = [super init]) {
+        self.segmentId = segmentId;
+        self.size = size;
+        self.start = start;
+        self.duration = duration;
+        self.frameCount = frameCount;
+        self.frameRate = frameRate;
+        self.height = height;
+        self.width = width;
+    }
+    return self;
+}
+
+- (nonnull NSArray<NSDictionary<NSString *, id> *> *)serialize
+{
+
+    long timestamp = [SentryDateUtil millisecondsSince1970:self.start];
+
+    NSDictionary *metaInfo = @{
+        @"type" : @4,
+        @"timestamp" : @(timestamp),
+        @"data" : @ { @"href" : @"", @"height" : @(self.height), @"width" : @(self.width) }
```

Comment:
`l`: Is `href` empty on purpose?

Reply:
Yes.  RRWebJson needs to have the property, but it can be empty